dev-libs/libunibreak/libunibreak-4.0.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="Line and word breaking library"
+DESCRIPTION="An implementation of the line breaking and word breaking \
+algorithms as described in Unicode Standard Annex 14 and Unicode Standard \
+Annex 29."
+HOMEPAGE="http://vimgadgets.sourceforge.net/libunibreak/"
+COPYRIGHT="2016 Andreas Röver"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/adah1972/libunibreak/archive/libunibreak_4_0.tar.gz"
+CHECKSUM_SHA256="8ea270d2d9dea6b633f7de1a4a624163c3549b4b6f5031f08c6b96d704d58aa4"
+SOURCE_DIR="libunibreak-libunibreak_4_0"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion=3.1.0
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libunibreak$secondaryArchSuffix = $portVersion
+	lib:libunibreak$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	"
+
+PROVIDES_devel="
+	libunibreak${secondaryArchSuffix}_devel = $portVersion
+	devel:libunibreak$secondaryArchSuffix = $libVersionCompat
+	devel:liblinebreak$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	libunibreak$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage libunibreak$secondaryArchSuffix \
+	"$libDir"/libunibreak.so.$libVersion
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+
+	# remove libtool library files
+	rm $libDir/*.la
+	rm $libDir/liblinebreak.a
+
+	# prepare develop/lib
+	prepareInstalledDevelLibs libunibreak
+	ln -s $developLibDir/libunibreak.a $developLibDir/liblinebreak.a
```

Comment:
```diff
-	ln -s $developLibDir/libunibreak.a $developLibDir/liblinebreak.a
+	ln -s libunibreak.a $developLibDir/liblinebreak.a
```
Alternatively, you could use `ln -r -s` and keep the same arguments.
BTW, maybe we don't need to symlink `libunibreak.a`.

Reply:
According my tests we do need the a symlink.

Reply:
OK.
(But it would still be nice to make it a _relative_ symlink.)